### ISSUE TITLE
Filter Properties that exist in the Custom Metadata Panel

### ISSUE THREAD

**Issue by Qrizzz** [adobe-dmeservices/custom-metadata#15]
I like to suggest a way of filtering for metadata in the Custom Metadata Panel. My usecase is that I like to be able to view only images with missing metadata. In the example data I like to find all pet images with missing breed. I like to save different filter settings for editors to find the images which need some work.

_Originally posted by @jameslockman in https://github.com/adobe-dmeservices/custom-metadata/issues/11#issuecomment-783455295_

**Comment by jameslockman** [comment]
Thanks, @Qrizzz . We'll take a look at what we might be able to provide in the context of the Custom Metadata Panel.

**Comment by jameslockman** [comment]
And to be clear, you would like to be able to select images with a specific property value and also items that lack a value for a property, correct?

**Comment by Qrizzz** [comment]
Exactly.I would benefit from an option to filter all images with a specific custom meta field that is blank, as well as a multi-filter option like "all female dogs, photographed in portrait orientation with the tag outdoor".

**Comment by jameslockman** [comment]
This is not forgotten. We have tried numerous ways to achieve this within what Bridge makes available, and we have come up dry. Would you be willing to accept a separate panel that provides expanded search capabilities tied to the Custom Metadata panel? Search results would appear within this new panel, not in the regular file browser, if so.

**Comment by jameslockman** [comment]
Please take a look at 2.021, which is off to Adobe for approval. We added a filter option that lets you choose one or more properties from your Views and set conditions for filtering. It starts at the current view and can look down in the tree, which takes more time, of course. The result is a new Collection with the relevant assets.
